Sources/Interaction/Style/InteractorStyleManipulator/index.js

```diff
@@ -420,13 +421,19 @@ function vtkInteractorStyleManipulator(publicAPI, model) {
       model.currentManipulator.getButton() === button
     ) {
       model.currentManipulator.onButtonUp(model._interactor);
-      model.currentManipulator.endInteraction();
-      model.currentManipulator = null;
-      model._interactor.cancelAnimation(publicAPI.onButtonDown);
+      const canEndManipulator = model.currentManipulator.endInteraction();
+      if (canEndManipulator) {
```

Comment:
let's face it, developers will forget to return something in "endInteraction()". 
Maybe `undefined` (when nothing is returned) should be considered as "ok to be ended".